ansible_collections/arista/avd/roles/eos_designs/python_modules/cloudvision_tags/avdstructuredconfig.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import annotations
+
+from functools import cached_property
+from typing import TYPE_CHECKING
+
+from ansible_collections.arista.avd.plugins.plugin_utils.avdfacts import AvdFacts
+from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.underlay.utils import UtilsMixin
```

Comment:
Please avoid importing from other modules. Move to shared utils if you still need something from this one.